<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SyntaxHighlightingSettings</class>
 <widget class="QWidget" name="SyntaxHighlightingSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Syntax Highlighting</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1,0,0">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>8-bit load/store/move instructions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="bit8lsmInstLabel">
       <property name="text">
        <string>MOV</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reset8lsm">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="system-reboot"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectColor8lsm">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,1,0,0">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Jump instructions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="jmpcallInstLabel">
       <property name="text">
        <string>JMP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetJMP">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="system-reboot"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectColorJMP">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,1,0,0">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>16-bt load/store/move instructions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="bit16lsmInstLabel">
       <property name="text">
        <string>SHLD</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reset16lsm">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="system-reboot"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectColor16lsm">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,1,0,0">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>8-bit arithmetic/logical instructions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="bit8alInstLabel">
       <property name="text">
        <string>ADD</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reset8al">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="system-reboot"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectColor8al">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,1,0,0">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>16-bit arithmetic/logical instructions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="bit16alInstLabel">
       <property name="text">
        <string>INX</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reset16al">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="system-reboot"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectColor16al">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,1,0,0">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Other</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="miscInstLabel">
       <property name="text">
        <string>HLT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetmiscInst">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="system-reboot"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectColormisc">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="2,1,0,0">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Pseudo instructions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pInstLabel">
       <property name="text">
        <string>STRING</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetpInst">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="system-reboot"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectColorpinst">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="2,1,0,0">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Comments</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="commentLabel">
       <property name="text">
        <string>//hello</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetcomm">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="system-reboot"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectColorcomm">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="2,1,0,0">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Characters</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="charsLabel">
       <property name="text">
        <string>&quot;hello word&quot;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetChars">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="view-refresh"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectColorChars">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="2,1,0,0">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Labels</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelsLabel">
       <property name="text">
        <string>label:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetLabels">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="system-reboot"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectColorLabels">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
